### PR TITLE
Updated the downloads pattern to match new look and feel. Added image…

### PR DESCRIPTION
**Pull request by trevorsaint**
Updated the look and feel of the **downloads component** to match new design. Added example thumbnails and PDF documents.
download-example

### CODE REVIEW

File: src/components/downloads/_downloads.scss [hunk 1]
```diff
@@ -1,35 +1,37 @@
 .download {
```
Comment (rmccar) [inline src/components/downloads/_downloads.scss:1]: In this file I would include `.download__image`, `.download__image-link`, `.download__meta` and `.download__excerpt` inside the `.download` brackets and then replace the "download" in each of them with "&". to fit in with the style of other SCSS files we have which use Sass. Other than that it looks good

**Comment by boxadesign** [comment]
Agree with Rich's comment other than that looks good!

File: src/components/downloads/_downloads.scss (hunk 1 above)
Comment (trevorsaint) [reply src/components/downloads/_downloads.scss:1]: Nested. Sure no problem.

**Comment by trevorsaint** [comment]
I am thinking it might be best to allow flexibility in the heading level.

I am using a H3 in this instance because it made sense for Resource for Partners. If being used elsewhere a H3 may not be suitable. What are your thoughts?

**Pull request merged**